include/fmt/format.h

```diff
@@ -3592,6 +3592,11 @@ template <typename Char> struct arg_formatter {
     // to the parse context.
     return out;
   }
+
+  static auto make_arg_formatter(iterator iter, format_specs<Char>& s)
+      -> arg_formatter {
+    return {iter, s, locale_ref()};
+  }
```

Comment:
Please move this function to `printf.h` where it is used (it doesn't need to be in `arg_formatter`) and add a comment that it is a workaround for the XL compiler bug, with a link to the bug report if possible.

Reply:
Done.

Unfortunately, the IBM ticket is not publicly accessible.